backend/services/GraphWebhookService.ts

```diff
@@ -0,0 +1,74 @@
+import { Client as GraphServiceWebhookClient } from '../types/openapi-account-service';
+import { OpenAPIClientAxios, type Document } from 'openapi-client-axios';
+import openapiJson from '../openapi-specs/account-service.json' assert { type: 'json' };
+import * as Config from '../config/config';
+import { HttpStatusCode } from 'axios';
+import logger from '../logger';
+import { HttpError } from '../types/HttpError';
+
+type GraphServiceWebhookResponse = {
+  referenceId: string;
+  msaId: string;
+  update: string;
+};
+
+type GraphChangeNotification = {
+  msaId: string;
+  update: string;
+};
```

Comment:
is there any way we can import these types directly from graph service?

Reply:
Possibly, this code is a placeholder for now. Some re-work is required in follow-on.